src/bse202/routes/user/recommendation.py

```diff
@@ -0,0 +1,40 @@
+from flask import Blueprint, jsonify
+from ...db import get_db
+
+user_blueprint = Blueprint("user", __name__)
```

Comment:
why?

```py
from .blueprint import user_blueprint
```

This is redundant, please check the code around it to see how to use it my friend